django-verdant/rca/static/rca/js/showcase-block.js

```diff
@@ -0,0 +1,26 @@
+$(function(){
+    // Showcase carousel
+    var $showcaseSlider = $('.js-showcase');
+        var prevButton = '<button aria-label="Previous" class="slider__button slider__button--prev" type="button"><svg class="slider__arrow slider__arrow--prev"><title>Previous</title><use xlink:href="#chevron"></use></svg></button>';
+        var nextButton = '<button aria-label="Next" class="slider__button slider__button--next" type="button"><svg class="slider__arrow slider__arrow--next"><title>Previous</title><use xlink:href="#chevron"></use></svg></button>';
+
+    $showcaseSlider.slick({
```

Comment:
I think we should only initiate the carousel if there is more than one testimonial - seems an element where there is likely to be only 1?